Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -355,7 +355,6 @@ import BraintreeDataCollector
         } else {
             apiClient.sendAnalyticsEvent(BTPayPalAnalytics.browserPresentationFailed)
             apiClient.sendAnalyticsEvent(BTPayPalAnalytics.tokenizeFailed)
-            completion(nil, BTPayPalError.webSessionFailedToLaunch)
```

Comment:
Wouldn't this now mean we are returning from this without a completion? We still want to send an error to the merchant here.

Reply:
I think @scannillo didn't want to return to merchant app at this point but for it to be handled in web session completion handler as an error.

Reply:
Gotcha, so this would now be returned in `completion(nil, BTPayPalError.webSessionError(error))` correct?